shared/js/actions.js

```diff
@@ -126,6 +126,14 @@ loop.shared.actions = (function() {
     }),
 
     /**
+     * Notifies that participant data has been received.
+     */
+    UpdatedParticipant: Action.define("updatedParticipant", {
+      name: String,
+      userId: String
```

Comment:
It'd be helpful to put the format/origin of the userId in the comments, even if we're currently just doing something hacky for bootstrapping.